app/helpers/bgeigie_imports_helper.rb

```diff
@@ -18,7 +18,7 @@ def bgeigie_nav_li(status) # rubocop:disable Metrics/AbcSize
                params[:by_status] == status.to_s
              end
     content_tag(:li, class: ('active' if active)) do
-      p = params.except(:action, :controller).merge(by_status: (status unless status == :all))
+      p = params.to_unsafe_h.except(:action, :controller).merge(by_status: (status unless status == :all))
```

Comment:
I realize that this will require deeper knowledge of the system, but, for security reasons, most likely we should be calling `.permit` somewhere and `to_h`, right?

https://stackoverflow.com/a/38859674

Reply:
Right. The goal of this PR is to have Rails 5 running. Later on, we can start refactoring because all these `*_nav_li` methods smell a little 👃 ;)